functorch/csrc/BatchRulesLinearAlgebra.cpp

```diff
@@ -151,6 +151,20 @@ Tensor linear_decomp(
   return result;
 }
 
+std::tuple<Tensor, c10::optional<int64_t>>
```

Comment:
Can we use `EXISTING_BDIM` or `EXISTING_BDIM_ALL_BOXED` for this?

Reply:
Unfortunately neither of them works. 
* `EXISTING_BDIM` seems to assume that only `self` has bdim 
* With `EXISTING_BDIM_ALL_BOXED` we get `torch.linalg.householder_product: input.shape[-1] must be greater than or equal to tau.shape[-1]`